feature/gribi/ate_tests/ack_in_presence_other_routes/route_ack_test.go

```diff
@@ -197,20 +197,16 @@ func configureNetworkInstance(t *testing.T) {
 }
 
 // configStaticRoute configures a static route.
-func configStaticRoute(t *testing.T, dut *ondatra.DUTDevice, prefix string, nexthop string) *telemetry.NetworkInstance_Protocol_Static {
-	d := &telemetry.Device{}
-	ni1 := d.GetOrCreateNetworkInstance(*deviations.DefaultNetworkInstance)
-	ni1.Enabled = ygot.Bool(true)
-	ni1.Name = ygot.String(*deviations.DefaultNetworkInstance)
-	ni1.Description = ygot.String("Static route added by gNMI-OC")
-	static := ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static-1")
+func configStaticRoute(t *testing.T, dut *ondatra.DUTDevice, prefix string, nexthop string) *telemetry.NetworkInstance_Protocol {
+	ni1 := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).Get(t)
+	static := ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "STATIC")
```

Comment:
Suggest protocol instance key names to be `DEFAULT` in the event there is only a single instance per protocol type.  This name is mostly meaningless (esp. in the case of static routes)

https://github.com/openconfig/public/blob/master/release/models/network-instance/openconfig-network-instance.yang#L1252-L1264

Reply:
Done! Thanks!